### DISCUSSION TITLE
Silent Installation

### DISCUSSION THREAD

**Discussion by Hardexit** [snappysnappydog/Simple-IP-Config#193] [Q&A] [answered]
Hello,

are there parameters for the installer if you want to distribute it automatically?

**Comment by KurtisLiggett** [comment]
The installer is built using Inno Setup. I haven't tested this, but so you should be able to do something like this:
`Simple.IP.Config.Setup.2.9.7.exe /SILENT`

[Silent install parameters](https://jrsoftware.org/ishelp/index.php?topic=setupcmdline)

**Reply by Hardexit** [reply, marked as answer]
Hi,

so with `/VERYSILENT` you can do the installation process completely in the background.
Thank you.
